Dockerfile

```diff
@@ -1,12 +1,28 @@
-FROM golang:1.10-alpine 
+FROM ubuntu:18.04
 
-RUN apk add --no-cache bash
+RUN apt-get update \
+ && apt-get install -y \
+                wget \
+                dpkg \
+                python \
+                golang \
+                git \
+                net-tools
 
-ADD . /go/src/github.com/pingcap/go-ycsb 
+RUN wget https://www.foundationdb.org/downloads/5.1.7/ubuntu/installers/foundationdb-clients_5.1.7-1_amd64.deb \
+ && dpkg -i \
+	foundationdb-clients_5.1.7-1_amd64.deb
 
-RUN cd /go/src/github.com/pingcap/go-ycsb && go build -o /go-ycsb ./cmd/* 
-RUN cp -rf /go/src/github.com/pingcap/go-ycsb/workloads /workloads
+ENV GOPATH /go
 
-# Use `docker exec -it go-ycsb bash` in another terminal to proceed.
-# hack for keep this container running
-CMD tail -f /dev/null
\ No newline at end of file
+RUN go get -u github.com/golang/dep/cmd/dep
+
+ADD . /go/src/github.com/pingcap/go-ycsb
+
+WORKDIR /go/src/github.com/pingcap/go-ycsb
+
+RUN /go/bin/dep ensure \
```

Comment:
seem it has already included vendor, why using ensure here again?

Reply:
I added `vendor` to `.dockerignore`. In this case we know that we use the same libraries as in lock file.

Reply:
@siddontang with this solution to build image takes a lot of time. I can switch back to copy vendors. If it would help.

Also I checked that image is 1GB. So I switched to multi stage build and now image's size is 180M